.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,45 @@
-Fixes: #
+## Description
 
-Explanation:  
+<!--
+This is a freeform area, but there are 3 questions you should try to answer:
 
-Manual testing steps:  
-  - 
-  - 
-  - 
\ No newline at end of file
+* What information do reviewers need to have in order to understand your changes?
+* What is the current state of things and why does it need to change?
+* What is the solution your changes offer?
+
+Below is a template to give you some ideas. Feel free to use your own words!
+
+(Hint: all of this should ideally go in your commit message so that people don't have to find this pull request in the future ;))
+-->
+
+Currently, ...
```

Comment:
Wary of including un-commented text like this in the templates because of how commonly users just leave it unaltered.

e.g. how many PRs have you seen with:

```
Manual testing steps:
- 
- 
- 
```

Bets on half the community PRs having the manual testing steps of "Go to this screen, do this, then do this"? :sweat_smile:

Reply:
Though we could instead agree as a team to start being strict about this. That's another option.

Reply:
Good call! As for whether we should be strict, I do want people to try and give more context around PRs as sometimes issues don't tell the full story but since everyone has their own style I'm trying not to completely dictate how people do that, just nudge them in a certain direction! But you're right that this should be designed for contributors as well. To that end I moved the prompts inside of the comments.